core/src/test/java/com/digitalpebble/stormcrawler/parse/filter/CollectionTaggerTest.java

```diff
@@ -12,7 +13,7 @@
     public void testCollectionTagger() throws IOException {
 
         CollectionTagger tagger = new CollectionTagger();
-        tagger.configure(new HashMap(), null);
+        tagger.configure(new HashMap<>(), NullNode.instance);
```

Comment:
Looks odd to me replacing `null` with some instance from jackson?

Reply:
That's a null node singleton and the tests always failed due to the fact, that the implementation wasn't supposed to consume null.

This was easier than changing the interface from `@NotNull` to `@Nullable`.

Reply:
Plus the nullcheck would be only needed for this test. Nowhere else the value becomes `null`.